interface/bigTree.h

```diff
@@ -180,6 +180,7 @@ public :
   std::vector<int>     *decayMode;
   std::vector<Long64_t> *tauID;
   std::vector<float>   *combreliso;
+  std::vector<float>   *tkRelIso;
```

Comment:
One additional comment: you should remove the following branches
```
-  std::vector<float>   *L1_tauEt;
-  std::vector<float>   *L1_tauEta;
-  std::vector<float>   *L1_tauPhi;
-  std::vector<short>   *L1_tauIso;
-  std::vector<float>   *L1_jetEt;
-  std::vector<float>   *L1_jetEta;
-  std::vector<float>   *L1_jetPhi;
```
because they are no longer part of bigNtuples.

Reply:
Done and committed.